docs/antrea-network-policy.md

```diff
@@ -255,8 +255,14 @@ default tier i.e. the "application" Tier.
 **ingress**: Each ClusterNetworkPolicy may consist of zero or more ordered
 set of ingress rules. Each rule, depending on the `action` field of the rule,
 allows or drops traffic which matches both the `from` and `ports` sections.
-Also, each rule has an optional `name` field, which is unique within the policy 
-describing the intention of this rule.
+Also, each rule has an optional `name` field, which should be unique within 
+the policy describing the intention of this rule. If `name` is not provided for 
+a rule, it will be auto-generated by Antrea. The auto-generated name will be 
+of format `[ingress/egress]-[action]-[uid]`, e.g.
+ingress-allow-2f0ed6ec1949367d0107ea931043df9eb430b1c3,
```

Comment:
could we use a shorter suffix? I think the chance of collision is small enough even we only use the first 6 chars (36^6=2176782336) given the name only needs to be unique within a policy and it has direction and action as prefix. (when generating network interface names for Pods, we do similar hash and use the first 6 chars as suffix). A short suffix looks cleaner and makes it possible to type it.

Reply:
Make sense. I can change it. 
BTW do we need an additional check for this collision or just assume it won't happen?

Reply:
we should do 7, the default short git sha length :)